File: netflipster/csharp/Netflix2/repository/DvdDao.cs
using System.Collections.Generic;

namespace Netflix2.repository
{
    public class DvdDao
    {
        public static IEnumerable<string> GetIdentifiers()
        {
            using (var cnx = DB.GetConnection())
            using (var rs = DB.ExecuteQuery("SELECT ID FROM DVDS WHERE ID > 0", cnx))
            {
                while (rs.Read())
                {
                    int id = (int)rs["ID"];
                    yield return id.ToString();
                }
            }
        }

        public static string GetDvdNameById(string id)
        {
            return DB.ExecuteScalar<string>($"SELECT NAME FROM DVDS WHERE ID = {id}");
        }
    }
}

File: netflipster/java/src/main/java/repository/DvdDao.java
package repository;

import source.DB;

import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.ArrayList;
import java.util.Collection;

public class DvdDao {

    /**
     * Get the list of DVDs identifiers
     *
     * @return list of DVD identifiers
     */
    public static Collection<String> getIdentifiers() {
        Collection<String> dvds = new ArrayList<>();
        ResultSet rs = DB.execute("SELECT ID FROM DVDS WHERE ID > 0");
        try {
            while (rs.next()) {
                int id = rs.getInt("ID");
                dvds.add(String.valueOf(id));
            }
        } catch (SQLException e) {
            e.printStackTrace();
        }
        return dvds;
    }

    /**
     * Get the name of the DVD
     *
     * @param id identifier of the DVD
     * @return list of DVD names
     */
    public static String getDvdNameById(String id) {
        Collection<String[]> dvds = new ArrayList<>();
        ResultSet rs = DB.execute(String.format("SELECT NAME FROM DVDS WHERE ID = %s", id));
        try {
            rs.next();
            return rs.getString("NAME");
        } catch (SQLException e) {
            e.printStackTrace();
        }
        return null;
    }
}
